app/assets/stylesheets/dashboards.scss

```diff
@@ -24,3 +24,28 @@
 .bootstrap_toggle_spacing .checkbox {
   margin-top: 0px;
 }
+
+.pt-phone,.pt-LMP {
+  width: 100px;
+  white-space: nowrap;
+  padding-right: 5px;
+}
+
+.pt-recent-notes {
```

Comment:
It's controlled by two things:

* in `app/models/Patient.rb`, around line 135, is how we control how many characters display in the note text:

```
  def most_recent_note_display_text
    note_text = most_recent_note.try(:full_text).to_s
    display_note = note_text[0..40]  #
    display_note << '...' if note_text.length > 41 # 
    display_note
  end
```

* The plus sign expander glyphicon comes from `app/helpers/notes_helper.rb` in the `plus_sign_glyphicon` method - the idea is that if a note's text is longer than `n`, it'll display the plus sign glyphicon. 

I think we could trim that back as far as we need to, since the plus sign glyphicon functionally works as an expander. How about trying to set `note_text[0..x]` to 15, `note_text.length > x` to 16, and the number in `plus_sign_glyphicon` to 16 and see how that looks?